README.md

```diff
@@ -408,11 +486,23 @@ javaCompileOptions {
 kapt {
   arguments {
     arg("deepLink.incremental", "true")
-    arg("deepLink.customAnnotations", "com.airbnb.AppDeepLink,com.airbnb.WebDeepLink")
+    arg("deepLink.customAnnotations", "com.airbnb.AppDeepLink|com.airbnb.WebDeepLink")
   }
 }
 ```
 
+**KSP**
+
+KSP is always incremental and you always have to provide the list of `deepLink.customAnnotation` if
+you have any or they will not be processed.
+
+```groovy
+ksp {
+  arg("deepLink.incremental", "true")
```

Comment:
I just wanted to keep it backwards compatible. It would imho not reduce complexity by that much. But it can be on the chopping block if it does not fit with any other major changes.